### ISSUE TITLE
Docker trust signing without correct permissions gives two errors 

### ISSUE THREAD

**Issue by eiais** [riyazdf/cli#68]
```
kyle@docker:~/go/src/github.com/docker/cli$ ./build/docker-linux-amd64 trust sign ubuntu:latest
Signing and pushing trust metadata for ubuntu:latest
Failed to sign "docker.io/library/ubuntu":latest - you are not authorized to perform this operation: server returned 401.
you are not authorized to perform this operation: server returned 401.
```

**Comment by eiais** [comment]
```
kyle@docker:~/go/src/github.com/docker/cli$ ./build/docker-linux-amd64 trust sign ubuntu/kyle:test
Signing and pushing trust metadata for ubuntu/kyle:test
you are not authorized to perform this operation: server returned 401.
```

**Comment by eiais** [comment]
```
kyle@docker:~/go/src/github.com/docker/cli$ ./build/docker-linux-amd64 trust sign ubuntu:latest
Signing and pushing trust metadata for ubuntu:latest
ERRO[0003] couldn't remove target from targets: could not find necessary signing keys, at least one of these keys must be available: 8273733f491f362bb36710fd8a99f78c3fbaecd8d09333985c76f1064b80760f 
Failed to sign "docker.io/library/ubuntu":latest - could not find necessary signing keys, at least one of these keys must be available: 8273733f491f362bb36710fd8a99f78c3fbaecd8d09333985c76f1064b80760f
Error: could not find signing keys for remote repository docker.io/library/ubuntu, or could not decrypt signing key: could not find necessary signing keys, at least one of these keys must be available: 8273733f491f362bb36710fd8a99f78c3fbaecd8d09333985c76f1064b80760f
```

EDIT:
This Error shows up if you have a remove in your changelist without the right keys

**Comment by riyazdf** [comment]
closed with #77
